docs/conf.py

```diff
@@ -182,32 +189,53 @@ def setup(app):
 # If true, `todo` and `todoList` produce output, else they produce nothing.
 todo_include_todos = True
 
+
 # -- Options for HTML output ----------------------------------------------
 
-# The theme to use for HTML and HTML Help pages.  See the documentation for
-# a list of builtin themes.
-#
-html_theme = "default"
+html_theme = "pydata_sphinx_theme"
+
+
+html_favicon = "_static/favicon.ico"
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
-#
-# html_theme_options = {}
+html_theme_options = {
+    "logo": {
+        "image_light": "ctapipe_logo.webp",
+        "image_dark": "ctapipe_logo_dark.webp",
+        "alt_text": "ctapipe",
+    },
+    "github_url": "https://github.com/cta-observatory/ctapipe",
+    "header_links_before_dropdown": 6,
+    "navbar_start": ["navbar-logo"],
+    "use_edit_page_button": True,
+    "icon_links": [
+        {
+            "name": "CTA Observatory",
+            "url": "https://www.cta-observatory.org/",
+            "type": "url",
+            "icon": "https://www.cta-observatory.org/wp-content/themes/ctao/favicon.ico",
+        },
+    ],
+}
+
+html_sidebars = {"**": ["sidebar-nav-bs.html", "sidebar-ethical-ads.html"]}
```

Comment:
This is a bit of a waste of space: there is no content in the sidebar for most of the pages

Reply:
The version switcher is there. We had an alternative version, but that would require manually curating a json file of versions.

Reply:
Yeah, you're right, but this is currently the only way to get the version switcher from RTD. The way I see it, we currently have three options:

1. We go back and reimplement the dropdown menu I previously added. This however would mean that we have to add any new version to a JSON file by hand.

2. We use the flyout menu from RTD, thus making it necessary to have the sidebar on all pages.

3. We use the flyout menu only on specific pages like the front page and for example the API docs.

I'm not sure which option is the best as I guess all of them have their drawbacks...

Reply:
Hmm, this works fine in the vodf pages I created.. Not sure how I configured it, but there the switcher is floating on the lower-right corner: https://vodf.readthedocs.io/en/latest/  (and also in the top-bar) 
![Uploading image.png…]()

Reply:
I tried to figure out why the behaviour is so different from your vodf pages today but couldn't find anything. My only guess would be different behaviour when using Sphinx 4.5 and PyData 0.10 for the vodf pages versus the ctapipe docs with Sphinx 6.2.1 and PyData 0.13.3...

Reply:
The version switcher in the navbar on the vodf pages is of course the same one that uses the manually curated json file. But the version switcher from RTD is what boggles my mind. 

At least from what I gathered from the [PyData theme dev docs ](https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/readthedocs.html#version-switcher) and the and an [open issue](https://github.com/pydata/pydata-sphinx-theme/issues/705), they are currently working on a way to make the RTD version switcher more customizable, or at least movable from the sidebar. So for now it should actually only appear in the primary sidebar (if not hidden) on the left.

Reply:
Who knows? I can try updating the VODF docs and see if it disappears... You're right that it's not quite the same version.